src/games/strategy/triplea/ai/proAI/util/ProMoveUtils.java

```diff
@@ -271,6 +272,39 @@ public static void calculateBombardMoveRoutes(final PlayerID player, final List<
     }
   }
 
+  public static void calculateBombingRoutes(final PlayerID player, final List<Collection<Unit>> moveUnits,
```

Comment:
`moveRoutes` is an out parameter, please remove it from the parameter list and return the calculated values.